openshift/ci-operator/source-image/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM src
 
 COPY oc /usr/bin/oc
 COPY --from=registry.ci.openshift.org/openshift/knative-v0.25.1:knative-serving-src /go/src/knative.dev/serving/ /go/src/knative.dev/serving/
-COPY --from=registry.ci.openshift.org/openshift/knative-v0.25.0:knative-eventing-src /go/src/knative.dev/eventing/ /go/src/knative.dev/eventing/
+COPY --from=registry.ci.openshift.org/openshift/knative-v0.25.1:knative-eventing-src /go/src/knative.dev/eventing/ /go/src/knative.dev/eventing/
```

Comment:
and here we do actually bump to the correct one.


```
docker pull registry.ci.openshift.org/openshift/knative-v0.25.1:knative-eventing-src
```

works - as expected

Reply:
This command does not work: 
```
podman pull registry.ci.openshift.org/openshift/knative-v0.25.0:knative-eventing-src
Trying to pull registry.ci.openshift.org/openshift/knative-v0.25.0:knative-eventing-src...
Error: Error initializing source docker://registry.ci.openshift.org/openshift/knative-v0.25.0:knative-eventing-src: Error reading manifest knative-eventing-src in registry.ci.openshift.org/openshift/knative-v0.25.0: manifest unknown: manifest unknown
```
So, I suppose the image was deleted but it's strange. Should not happen that quickly. IMO
I'd suggest contacting #forum-testplatform.
Once this current PR is merged this section should be automatically updated: https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.8.yaml#L10 , and it should no longer require 0.25.0

Reply:
Do I need to manually https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.8.yaml#L10  patch???  I _think_ (not 100% sure) we have never done that though, when bumping image promotion... :disappointed:

Reply:
@mgencur added a PR: https://github.com/openshift/release/pull/22533